apps/frontend/src/components/SendForm.svelte

```diff
@@ -1,17 +1,19 @@
 <script lang="ts">
+  import { getContext } from 'svelte';
+  import { Chain, CreateTxReturn } from '@baf-wallet/interfaces';
   import Button from './base/Button.svelte';
-  import Input from './base/Input.svelte';
-  import InputNumeric from './base/InputNumeric.svelte';
   import TxModal from './TxModal.svelte';
+  import SendNearFormPart from './near/SendNearFormPart.svelte';
+  import { AllBuildTxParams } from '@baf-wallet/multi-chain'
 
-  let sendAddr: string, amount: number;
+  let createTX: () => Promise<CreateTxReturn<AllBuildTxParams>>; //ChainTransaction
```

Comment:
perhaps instead of a union type we can use `<T>() => Promise<CreateTxReturn<InferBuildParams<T>>>` or something like this?